es/index.js

```diff
@@ -116,6 +116,9 @@ function fixConfig(axios, config) {
   if (axios.defaults.httpsAgent === config.httpsAgent) {
     delete config.httpsAgent;
   }
+  if (config.baseURL) {
+    config.url = config.url.replace(config.baseURL, '');
```

Comment:
Its more of a workaround really.. As others are commenting on issue #67 , axios really expects an **absolute** url here.

This is also an expectation in the core code of Axios (there are `isAbsoluteUrl` checks before `baseURL` gets applied to anything).

So, the problem is that by using a relative url in `baseURL`, it breaks axios's current expectations.

Reply:
Indeed, it does not look like an axios-retry issue.